src/Views/BatteryView.vala

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2011-2021 elementary, Inc. (https://elementary.io)
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA.
+ */
+ public class Power.BatteryView : Granite.SimpleSettingsPage {
+    private Battery battery;
+    //  private Services.DeviceManager dm;
+
+    public BatteryView (Gtk.Widget icon) {
+        Object (
+            header: _("Devices"),
+            title: _("Built-in"),
+            display_widget: icon,
+            description: _("Rechargeable batteries naturally lose capacity over time and when used. To maximize battery health, avoid leaving your device connected to power after it is charged.")
+        );
+    }
+
+    construct {
+      battery = new Battery ();
+      status = battery.get_info ();
+      content_area.row_spacing = 6;
+      content_area.margin_left = 50;
+      content_area.margin_top = 20;
```

Comment:
Two things here:

1. Instead of `margin_left`, we use `margin_start` because of right-to-left languages, but
2. I think what we actually want here is to center this within the horizontal space (and leave the default top margin):

```suggestion
      content_area.halign = Gtk.Align.CENTER;
```